.github/workflows/docker.yml

```
@@ -49,7 +49,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        target: [ gnoland-slim, gnokey-slim, gno-slim, gnofaucet-slim, gnoweb-slim ]
+        target:
+          [gnoland-slim, gnokey-slim, gno-slim, gnofaucet-slim, gnoweb-slim]
```

```suggestion
        target:
          - gnoland-slim
          - gnokey-slim
          - gno-slim
          - gnofaucet-slim
          - gnoweb-slim
```

Also cc @ajnavarro for his work on CI revamp.